Test fails

PaginationBoxView-test.js fails:

```
➜ make test

> react-paginate@0.5.2 test /home/user1/react-paginate
> BABEL_JEST_STAGE=0 jest

Using Jest CLI v0.8.2, jasmine1
 FAIL  __tests__/PaginationBoxView-test.js 
● Runtime Error
SyntaxError: /home/user1/react-paginate/__tests__/PaginationBoxView-test.js: Unexpected token (11:4)
npm ERR! Test failed.  See above for more details.
make: *** [test] Błąd 1
```

My env:
- node - 5.5.0
- npm - 3.3.12

I got the same error while building on travis: https://travis-ci.org/platan/react-paginate/jobs/107331414

--- a/.babelrc
+++ b/.babelrc
@@ -1,6 +1,7 @@
 {
   "presets": [
     "react",
-    "es2015"
+    "es2015",
+    "stage-0"
   ]
 }

--- a/README.md
+++ b/README.md
@@ -38,6 +38,10 @@ $ cd react-paginate
 
 Install dependencies:
 
+```console
+$ make install
+```
+
 Prepare the demo:
 
 ```console

--- a/__tests__/PaginationBoxView-test.js
+++ b/__tests__/PaginationBoxView-test.js
@@ -25,7 +25,7 @@ describe('PaginationBoxView', () => {
     expect(ReactDOM.findDOMNode(pagination).querySelector(".selected a").textContent).toBe("1");
 
     const pages = ReactDOM.findDOMNode(pagination).querySelectorAll("li");
-    expect(pages.length).toEqual(8);
+    expect(pages.length).toEqual(9);
   });
 
   it('test previous and next buttons', () => {
@@ -58,6 +58,6 @@ describe('PaginationBoxView', () => {
     );
     const pageItems = ReactDOM.findDOMNode(smallPagination).querySelectorAll("li");
     // Prev, current, next
-    expect(pageItems.length).toBe(3);
+    expect(pageItems.length).toBe(4);
   });
 });

--- a/__tests__/PaginationBoxView-test.js
+++ b/__tests__/PaginationBoxView-test.js
@@ -1,8 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import PaginationBoxView from './../react_components/PaginationBoxView';
-import PaginationListView from './../react_components/PaginationListView';
-import PageView from './../react_components/PageView';
+
+jest.dontMock('./../react_components/PaginationBoxView');
+jest.dontMock('./../react_components/PaginationListView');
+jest.dontMock('./../react_components/PageView');
+
+const PaginationBoxView = require('./../react_components/PaginationBoxView').default;
+const PaginationListView = require('./../react_components/PaginationListView').default;
+const PageView = require('./../react_components/PageView').default;
+
 import ReactTestUtils from 'react-addons-test-utils';
 
 

